hack/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: build push
+
+IMAGE_NAME := quay.io/app-sre/fleet-manager
```

Comment:
@msmagnanijr Do you need to change the image tag to be specific to our project e.g quay.io/app-sre/rhose-fleet-manager ?

Reply:
@robpblake it makes sense. Let me update it :)

Reply:
just to be clear, our images are required to be in org appsre no quay.